credsweeper/file_handler/file_path_extractor.py

```diff
@@ -130,7 +130,10 @@ def check_exclude_file(cls, config: Config, path: str) -> bool:
             return True
         if any(exclude_path in path for exclude_path in config.exclude_paths):
             return True
-        if Util.get_extension(path) in config.exclude_extensions:
+        file_extension = Util.get_extension(path)
+        if file_extension in config.exclude_extensions:
+            return True
+        if not config.depth and file_extension in config.exclude_containers:
             return True
```

Comment:
And here, is there any reason to just compare `file_extension` without using `lower()`?
In my opinion we can use and manage all extensions with lower case.

Reply:
lower() was applied for ``path`` in the function(line#126), so there is some reducing of double_work.